docs/content/docs/install/operator_installation.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Installation Through Operator
+weight: 2.1
+---
+# Installation Through Operator
+
+The easiest way to install Pipelines as Code on OpenShift is with the [Red Hat Openshift Pipelines Operator](https://docs.openshift.com/container-platform/latest/cicd/pipelines/installing-pipelines.html).
+
+On the OpenShift Pipelines Operator, the default namespace is `openshift-pipelines`.
+
+**Note:**
+
+When Pipelines as Code is installed through the [Tekton Operator](https://github.com/tektoncd/operator) the configurations of Pipelines as Code is
```

Comment:
does it work with the upstream operator?

Reply:
Yes upstream Operator only for Openshift Target